src/main/java/com/notably/view/SideBarTreeView.java

```diff
@@ -127,4 +134,27 @@ private void setSelectedProperty(Block block) {
             }
         }
     }
+
+    /**
+     * Custom {@code EventDispatcher} to allow for finer control over mouse click events.
+     * A {@link SideBarTreeViewCell} should have the flexibility to expand or contract through
+     * user mouse clicks, but should not respond to any other mouse events.
```

Comment:
You might wanna update the Javadoc here since there's no expanding event any more with the new changes

Reply:
ohh good catch!